scripts/nmt/nmt.py

```diff
@@ -75,6 +75,8 @@ def request(inputs,args):
                 result = {'msg': 'already exists error'}
             elif e.code() == grpc.StatusCode.UNAVAILABLE:
                 result = {'msg': 'server unavailable check network'}
+            else:
+                result = {'msg': 'unkown error'}
```

Comment:
instead of unknown error, will it be good to print the error code? so the user can identify what's the error from that.

Reply:
there can be several possible errors here as per this https://grpc.github.io/grpc/core/md_doc_statuscodes.html so better to print that instead of unknown.